test/e2e/action-group.test.js

```diff
@@ -9,10 +8,49 @@ describe('vl-action-group', async () => {
         return vlActionGroupPage.load();
     });
 
-    after((done) => { 
-        if (driver) {
-            driver.quit();
-        }
-        done();
+    it('als gebruiker kan ik het verschil zien tussen een left, center en right aligned action group', async () => {
+        const actionGroup = await vlActionGroupPage.getActionGroup();
+        const actionGroupAlignCenter = await vlActionGroupPage.getAlginCenterActionGroup();
+        const actionGroupAlignRight = await vlActionGroupPage.getAlignRightActionGroup();
+
+        await assert.eventually.isTrue(actionGroup.isLeftAligned());
+        await assert.eventually.isFalse(actionGroupAlignCenter.isLeftAligned());
+        await assert.eventually.isFalse(actionGroupAlignRight.isLeftAligned());
+
+        await assert.eventually.isFalse(actionGroup.isCenterAligned());
+        await assert.eventually.isTrue(actionGroupAlignCenter.isCenterAligned());
+        await assert.eventually.isFalse(actionGroupAlignRight.isCenterAligned());
+
+        await assert.eventually.isFalse(actionGroup.isRightAligned());
+        await assert.eventually.isFalse(actionGroupAlignCenter.isRightAligned());
+        await assert.eventually.isTrue(actionGroupAlignRight.isRightAligned());
+    });
+
+    it('als gebruiker kan ik het verschil zien tussen een standaard en een action group space-between variant', async () => {
+        const actionGroup = await vlActionGroupPage.getActionGroup();
+        const actionGroupSpaceBetween = await vlActionGroupPage.getSpaceBetweenActionGroup();
+
+        await assert.eventually.isFalse(actionGroup.hasSpaceBetween());
+        await assert.eventually.isTrue(actionGroupSpaceBetween.hasSpaceBetween());
     });
+
+    it('als gebruiker kan ik het verschil zien tussen een standaard en een action group bordered variant', async () => {
+        const actionGroup = await vlActionGroupPage.getActionGroup();
+        const actionGroupBordered = await vlActionGroupPage.getBorderedActionGroup();
+
+        await assert.eventually.isFalse(actionGroup.isBordered());
+        await assert.eventually.isTrue(actionGroupBordered.isBordered());
+    });
+
+    it('als gebruiker kan ik op de knoppen klikken van een action group', async () => {
+        const actionGroup = await vlActionGroupPage.getActionGroup();
+        const primaryButton = await actionGroup.getButtonMetIndex(0);
+        const secondaryButton = await actionGroup.getButtonMetIndex(1);
+        await primaryButton.click();
+        await secondaryButton.click();
+        await assert.eventually.isFalse(primaryButton.isSecondary());
+        await assert.eventually.isTrue(secondaryButton.isSecondary());
```

Comment:
maakt action-group automatisch den 1e primary of is dit gewoon door inhoud van de action-group? want in het 2e geval doet dat niet terzake hier.

Reply:
Is inderdaad door de slots die in de demo pagina zitten, doet in dit geval niet veel terzake, maar ik wou hier controleren dat het page object wel degelijk een VlButton terug geeft en daar niet blind op vertrouwen.